tests/foreman/cli/test_repository.py

```diff
@@ -784,7 +820,33 @@ def test_positive_synchronize_yum_repo(self):
                 self.assertEqual(new_repo['sync']['status'], 'Success')
 
     @run_only_on('sat')
-    @skip_if_bug_open('bugzilla', 1328092)
+    @tier1
+    def test_positive_synchronize_file_repo(self):
+        """Check if repository can be created and synced
+
+        :id: eafc421d-153e-41e1-afbd-938e556ef827
+
+        :expectedresults: Repository is created and synced
+
+        :CaseLevel: Integration
+        """
+        new_repo = self._make_repository({
+            u'content-type': u'file',
+            u'url': CUSTOM_FILE_REPO,
+        })
+        # Assertion that repo is not yet synced
+        self.assertEqual(new_repo['sync']['status'], 'Not Synced')
+        # Synchronize it
+        Repository.synchronize({'id': new_repo['id']})
+        # Verify it has finished
+        new_repo = Repository.info({'id': new_repo['id']})
+        self.assertEqual(new_repo['sync']['status'], 'Success')
+        self.assertEqual(
+            new_repo['content-counts']['files'],
+            str(CUSTOM_FILE_REPO_FILES_COUNT)
```

Comment:
And if tomorrow devs fix output and files count becomes int value? Better solve it the opposite way:
```python
self.assertEqual(
    int(new_repo['content-counts']['files']),
    CUSTOM_FILE_REPO_FILES_COUNT
)
```